container/worker/run_openqa_worker.sh

```diff
@@ -9,7 +9,9 @@ fi
 /root/qemu/kvm-mknod.sh
 
 if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
-  groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
+  group=$(ls -lhn /dev/kvm | cut -d ' ' -f 4)
+  [[ $(getent group kvm | cut -d ":" -f3) -ne "$group" ]] && groupmod -g "$group" --non-unique kvm
+  [[ $(getent group kvm | cut -d ":" -f3) -ne "$group" ]] && { echo "Error: The kvm group id must be 0"; exit 1; } 
```

Comment:
The error message makes so sense. How many times have I already written that this code sets the ID of the `kvm` group within the container to the group ID of the mapped `/dev/kvm` so both match? Hence the group id is *not* necessarily supposed to be 0. A better error message would be "Unable to ensure the ID of the kvm matches the group ID of /dev/kvm. KVM will not work.".